clojure-ts-mode.el

```diff
@@ -2479,7 +2479,9 @@ This function checks if `clojure-ts-mode' is compatible with the
 currently installed grammar.  The simplest way to do this is to validate
 a query that is valid in a previous grammar version but invalid in the
 required version."
-  (treesit-query-valid-p 'clojure '((sym_lit (meta_lit)))))
+  (ignore-errors
```

Comment:
This doesn't look like inlining to me. :D

Reply:
Would you prefer to define a separate function?

I just replaced the `treesit-query-valid-p` call with it's implementation.